src/config/index.js

```diff
@@ -1,8 +1,7 @@
-import getConfig from "next/config";
 import { getLogger } from "../utils/logger";
 
 const { trace } = getLogger("config");
-const { publicRuntimeConfig } = getConfig();
+const publicRuntimeConfig = { network: process.env.NET };
```

Comment:
wonder why this is named publicRuntimeConfig, afaik this term is a nextjs term

Reply:
so just config ?

Reply:
its weird i'm not sure what's up with the object structure and all - can check with @yehjxraymond ?

Reply:
Yep its a nextjs thingy. Config will do

Reply:
done